pkg/scheduler/validation/validation_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Move this file to `pkg/scheduler/apis/config/validation`

Reply:
This file has been deleted.